apps/emqx_authn/src/simple_authn/emqx_authn_jwt.erl

```diff
@@ -384,25 +384,47 @@ verify_claims(Claims, VerifyClaims0) ->
     Now = os:system_time(seconds),
     VerifyClaims =
         [
-            {<<"exp">>, fun(ExpireTime) ->
-                is_integer(ExpireTime) andalso Now < ExpireTime
-            end},
-            {<<"iat">>, fun(IssueAt) ->
-                is_integer(IssueAt) andalso IssueAt =< Now
-            end},
-            {<<"nbf">>, fun(NotBefore) ->
-                is_integer(NotBefore) andalso NotBefore =< Now
-            end}
+            {<<"exp">>, true,
```

Comment:
Suggestion: maybe we could use `required | optional` instead of booleans to avoid boolean blindness. :smile_cat:

Reply:
✅